understanding/target-size-enhanced.html

```diff
@@ -56,144 +56,125 @@
-               <h2>Intent</h2>
-               <p>The intent of this success criterion is to help users who may have trouble activating
-                  a small target because of hand tremors, limited dexterity or other reasons. If the
-                  target is too small, it may be difficult to aim at the target. Mice and similar pointing
-                  devices can be hard to use for these users, and a larger target will help them greatly
-                  in having positive outcomes on the web page.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、手の震え、器用さの限界、又はその他の理由により、小さなターゲットをアクティブにすることが困難な利用者を支援することである。ターゲットが小さすぎると、ターゲットを狙うのが困難になる場合がある。マウス及び類似のポインティングデバイスは、これらの利用者にとって使用するのが困難な場合があり、より大きなターゲットは、ウェブページで良い結果を得るのに大いに役立つ。
                </p>
-               <p>Touch is particularly problematic as it is an input mechanism with coarse precision.
-                  Users lack the same level of fine control as on inputs such as a mouse or stylus.
-                  A finger is larger than a mouse pointer, and generally obstructs the user's view of
-                  the precise location on the screen that is being touched/activated.
+               <p>タッチは、精度の粗い入力メカニズムであるため、特に問題がある。利用者は、マウス、スタイラスペンなどの入力と同じレベルの細かい制御ができない。指はマウスポインタよりも大きいため、通常、タッチ又はアクティブ化されている画面上の正確な位置を利用者が確認できなくなる。
                </p>
-               <p>The issue can be further complicated for responsive/mobile sites which need to accommodate
-                  different types of fine and coarse inputs (e.g. a site that can be accessed both on
-                  a traditional desktop/laptop with a mouse, as well as on a tablet or mobile phone
-                  with a touch screen).
+               <p>この問題は、異なる種類の細かい入力及び粗い入力に対応する必要があるレスポンシブ又はモバイルサイト (例えば、マウスを使用する従来のデスクトップ又はラップトップと、タッチスクリーンを使用するタブレット又は携帯電話との両方でアクセスできるサイト) では、さらに複雑になる可能性がある。
                </p>
-               <p>While this criterion defines a minimum target size, it is recommended that larger
-                  sizes are used to reduce the possibility of unintentional actions. This is particularly
-                  relevant if any of the following are true:
+               <p>この基準では最低限のターゲットサイズを定義するが、意図しないアクションの可能性を減らすために、より大きなサイズを使用することを勧める。これは、次のいずれかに該当する場合に特に重要である:
                </p>
                <ul>
                   			  
-                  <li>the control is used frequently;</li>
+                  <li>コントロールは頻繁に使用される。</li>
                   			  
-                  <li>the result of the interaction cannot be easily undone;</li>
+                  <li>インタラクションの結果は簡単に元に戻すことができない。</li>
                   			  
-                  <li>the control is positioned where it will be difficult to reach, or is near the edge
-                     of the screen;
+                  <li>コントロールは手の届きにくい場所、つまり画面の端の近くに配置されている。
                   </li>
                   			  
-                  <li>the control is part of a sequential task.</li>
+                  <li>コントロールは連続するタスクの一部である。</li>
```

Comment:
sequential の訳が悩ましいところですがまあ置いておくとして 

```suggestion
                  <li>コントロールが連続するタスクの一部である。</li>
```